.changelog/unreleased/bug-fixes/4367-fix-bench-load-validators.md

```diff
@@ -0,0 +1,2 @@
+- BenchmarkLoadValidators no longer panics
```

Comment:
don't think we need a changelog for this

Reply:
should I delete it then?

Reply:
I'd consider this changelog user-facing (for devs using Comet and ops). We don't need it because Git stores all the history for us.

Reply:
> should I delete it?

yes please